RelVal/o2dpg_release_validation.py

```diff
@@ -336,7 +336,8 @@ def plot_values_thresholds(summary, out_dir, title, include_patterns=None, exclu
     test_histo_value_map = extract_from_summary(summary, ["value", "threshold"], include_patterns, exclude_patterns)
 
     for which_test, histos_values_thresolds in test_histo_value_map.items():
-
+        if which_test == "summary":
```

Comment:
can we change it to compare to the global variable?
https://github.com/AliceO2Group/O2DPG/blob/master/RelVal/o2dpg_release_validation.py#L120

Same below

Reply:
OK, done